src/atomate2/forcefields/flows/mpmorph.py

```diff
@@ -0,0 +1,760 @@
+            "quench_start_temperature": 3000,
+        }
+    """
+
+    name: str = "MP Morph MLFF MD Maker"
+    temperature: float = 300
+    steps_convergence: int = 5000
+    steps_total_production: int = 10000
+
+    md_maker: ForceFieldMDMaker | None = field(default_factory=ForceFieldMDMaker)
+    convergence_md_maker: EquilibriumVolumeMaker | None = None
+    production_md_maker: ForceFieldMDMaker = field(default_factory=ForceFieldMDMaker)
+
+    quench_maker: FastQuenchMLFFMDMaker | SlowQuenchMLFFMDMaker | None = None
+    quench_maker_kwargs: dict[str, Any] | None = None
+
+    def _post_init_update(self) -> None:
+        """Ensure that forcefield makers correctly set temperature."""
+        self.md_maker = self.md_maker.update_kwargs(
+            update={
+                "temperature": self.temperature,
+                "n_steps": self.steps_convergence,
+            },
+            class_filter=ForceFieldMDMaker,
+        )
+        self.convergence_md_maker = EquilibriumVolumeMaker(
+            name="MP Morph MLFF Equilibrium Volume Maker",
+            md_maker=self.md_maker,
+            postprocessor=MPMorphEVPostProcess(),
+        )  # TODO: check EV versus PV
+
+        self.production_md_maker = self.md_maker.update_kwargs(
+            update=dict(
+                name="Production Run MLFF MD Maker",
+                temperature=self.temperature,
+                n_steps=self.steps_total_production,
+            )
+        )
+
+        self.quench_maker_kwargs = self.quench_maker_kwargs or {}
+        if len(self.quench_maker_kwargs) > 0:
+            self.quench_maker = self.quench_maker.update_kwargs(
+                update=self.quench_maker_kwargs, class_filter=type(self.quench_maker)
+            )
+
+
+@dataclass
+class SlowQuenchMLFFMDMaker(SlowQuenchMaker):
+    """Slow quench from high to low temperature using ForceFieldMDMaker.
+
+    Quenches a provided structure with a molecular dynamics run
+    from a desired high temperature to a desired low temperature.
+    Flow creates a series of MD runs that holds at a certain temperature
+    and initiates the following MD run at a lower temperature (step-wise
+    temperature MD runs).
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    md_maker :  ForceFieldMDMaker
+        MDMaker to generate the molecular dynamics jobs specifically for MLFF MDs
+    quench_start_temperature : float = 3000
+        Starting temperature for quench; default 3000K
+    quench_end_temperature : float = 500
+        Ending temperature for quench; default 500K
+    quench_temperature_step : float = 500
+        Temperature step for quench; default 500K drop
+    quench_n_steps : int = 1000
+        Number of steps for quench; default 1000 steps
+    """
+
+    name: str = "ForceField slow quench"
+    md_maker: ForceFieldMDMaker = field(default_factory=ForceFieldMDMaker)
+
+    def call_md_maker(
```

Comment:
I think so - the VASP and forcefield MD ensemble attributes ($N$, $V$, $T$, and $p$) are set in different locations (the `VaspInputSetGenerator` vs. just top-level `Maker` kwargs) so this might have to stay. Open to suggestions